pyccel/codegen/printing/pycode.py

```diff
@@ -250,7 +251,7 @@ def _print_FunctionDefArgument(self, expr):
         default = ''
 
         if expr.annotation:
-            type_annotation = self._print(expr.annotation)
+            type_annotation = f"'{self._print(expr.annotation)}'"
```

Comment:
Sorry, I do not understand this change. `self._print` should always return a string, right?

Reply:
Yes but the string is not necessarily quoted. Note the `"'` where `"` denotes a string and `'` will appear in the string `type_annotation`

Reply:
Previously I was letting `_print_SyntacticTypeAnnotation` but this is unhelpful if you need to combine it with `_print_UnionTypeAnnotation`, `_print_FunctionTypeAnnotation` or `_print_TypingFinal`

Reply:
I looked at it many times but my brain could not see the quotes! 😅